marathon.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "/kafka-manager",
+  "cpus": 0.5,
+  "mem": 256.0,
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "stealthly/kafkamanager",
```

Comment:
`"image": "stealthly/kafka-manager",`
instead of
`"image": "stealthly/kafkamanager",`